Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'govuk_app_config', '~> 1.2'
 gem 'nokogiri', '~> 1.8'
 gem 'notifications-ruby-client', '~> 2.6'
 gem 'plek', '~> 2.1'
+gem 'redcarpet'
```

Comment:
Should we specify a version here? `~> 3.4`?

Reply:
I was going to do that but didn't we decide pinning was pointless in our new dependabot world?

Reply:
Yeah, it is. Just makes it inconsistent with the others... I'm fine either way :man_shrugging:

Reply:
Cool... I think I'll leave it. We should unpin as we upgrade versions I suppose if that's what we have decided to do.

Reply:
Still useful for bundle updates, dependabot will also maintain the semvar pinning too.

Reply:
Agh... I'll pin it then.